test-kotlin/src/com/intellij/rt/coverage/runner.kt

```diff
@@ -29,13 +29,62 @@ fun runWithCoverage(coverageDataFile: File, testName: String, sampling: Boolean,
     return CoverageStatusTest.runCoverage(classPath, coverageDataFile, "kotlinTestData.*", "kotlinTestData.$testName.Test", sampling, calcUnloaded)
 }
 
-internal fun assertEqualsLines(project: ProjectData, expectedLines: Map<Int, String>, classNames: List<String>) {
+internal val allNames = listOf("ALL CLASSES")
```

Comment:
2 constants with similar values in different files, could you please group them somewhere?